default/generated/technology-usage/194-connect.windup.yaml

```diff
@@ -162,14 +174,10 @@
   description: Embedded library - RocketMQ Client
   labels:
   - konveyor.io/include=always
-  links:
-  - title: Red Hat JBoss Enterprise Application Platform (EAP) 6 Supported Configurations
-    url: https://access.redhat.com/articles/111663
-  - title: Red Hat JBoss Enterprise Application Platform (EAP) 7 Supported Configurations
-    url: https://access.redhat.com/articles/2026253
+  links: []
   ruleID: connect-02400
   tag:
-  - RocketMQ Client
+  - ' Client'
```

Comment:
Same here

Reply:
> Same here

It is the same as the previous one, https://github.com/konveyor/rulesets/blob/e1ca097b35d990cab32067b55ad1144d37d10d5c/default/generated/technology-usage/251-connect.groovy.windup.yaml#L139C1-L140C14 also copied from the upstream